netflix-roulette/src/components/AppHome/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { StyledButton, StyledTextField } from "./SearchBar.styles";
+import { Box } from "@material-ui/core";
+
+interface CounterProps {
+  handleSearch: (searchValue: string) => void;
+}
+
+interface CounterState {
+  searchValue: string;
+}
+
+export class SearchBar extends React.Component<CounterProps, CounterState> {
+  state = { searchValue: "" };
+
+  handleInputChange(event: any) {
```

Comment:
event type should be **React.ChangeEvent<HTMLInputElement>** but not **any**

Reply:
Updated, thanks